third_party/terraform/resources/resource_cloudiot_registry.go

```diff
@@ -369,3 +471,79 @@ func resourceCloudIoTRegistryStateImporter(d *schema.ResourceData, meta interfac
 	d.SetId(id)
 	return []*schema.ResourceData{d}, nil
 }
+
+func resourceCloudIotRegistryStateUpgradeV0toV1(rawState map[string]interface{}, meta interface{}) (map[string]interface{}, error) {
+	log.Printf("[DEBUG] State before upgrade: %#v", rawState)
+
+	oldCfg, ok := rawState["event_notification_config"]
+	if ok {
+		delete(rawState, "event_notification_config")
```

Comment:
What does deleting the value in state do? Won't we immediately add it back in `Read`?